api/swagger.yml

```diff
@@ -637,6 +637,7 @@ paths:
       description: >-
         Get transaction from the pool by its ID. This endpoint is specially useful for tracking the status of a transaction that may not be forged yet.
         Only transactions from the pool will be returned.
+        There are other options to filter transactions using query string parameters. Please check more details below.
```

Comment:
I was reviewing it with @jleebee and she told me we need to change the last sentence to
```Please check for more details below```